rfcs/tentative_testdriver_methods.md

```diff
@@ -0,0 +1,29 @@
+# RFC 226: Tentative testdriver methods
+
+## Summary
+Allow tentative methods to be added to testdriver.js that are not yet included in the WebDriver or WebDriver BiDi specifications.
+This should only be allowed where there is consensus between multiple browser vendors and where there is clear progress towards a specification.
+
+## Background
+As part of the [Interop 2025 Accessibility Investigation Area](https://github.com/web-platform-tests/interop-accessibility/issues/148), the intention is to extend browsers and WPT to allow additional accessibility properties to be tested by web platform tests.
+However, there are open questions regarding the shape of the API for exposing these properties, what properties should be exposed, how the tests should be written, etc.
+While the end goal is to extend the WebDriver specification with the required new endpoints, these open questions need to be answered before this is feasible.
+To answer these questions, it would be helpful if browser vendors could collaborate on a tentative but working implementation.
```

Comment:
For things like the Accessibility Investigation, is there any intention for there to be any, even high-level, explainer of what the proposed endpoints are? I'm mostly concerned about the risk of ending up somewhere where it is hard for anyone else to start passing the tests written using the tentative endpoint.

Reply:
One open question here is whether we need this RFC at all or whether an explainer is sufficient. See https://github.com/web-platform-tests/rfcs/pull/226#discussion_r2224692810. If an explainer is sufficient, is it sufficient for that explainer to be a GitHub issue (as is the case now) or does it need to be a document (to make it easier to follow)?